### ISSUE TITLE
A11Y issue - aria-selected applied incorrectly

### ISSUE THREAD

**Issue by rachelhoffmanapree** [jackocnr/intl-tel-input#2113]
Issue: aria-selected attribute is applied to highlighted list item rather than selected list item

In screenshot below you can see that the current user-selected list item is Argentina but aria-selected is set to true on Canada

<img width="1197" height="544" alt="Image" src="https://github.com/user-attachments/assets/df1c9d18-408c-4921-b585-0c1a19b67d3d" />

Root cause is that the aria-selected attribute is currently being set in the `highlightListItem` method when it should be set in the `selectListItem` method

**Comment by jackocnr** [comment]
Are you using the latest version?

**Comment by rachelhoffmanapree** [comment]
We're using version 25.3.0!

**Comment by jackocnr** [comment]
Thanks for raising this. This should be fixed in v25.13.2.

**Comment by rachelhoffmanapree** [comment]
Just updated to that version and it is indeed fixed! Thank you so much @jackocnr :)
